core/crypto/nike/hybrid/hybrid.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/katzenpost/katzenpost/core/crypto/rand"
 )
 
-var NOBS_CSIDHX25519 nike.Scheme = &scheme{
+var NOBS_CSIDH512X25519 nike.Scheme = &scheme{
 	name:   "NOBS_CSIDH-X25519",
```

Comment:
Should this name be `NOBS_CSIDH512-X25519`? I don't understand the relation between the variable name and the scheme `name` parameter in these. In `ctidh.go` you updated all three to have the same name

Reply:
the name string allows you to get the schema with a query with that string name from the "kem/schemes" module